### PR TITLE
Update documentation and config with start_sequence 

### PR DESCRIPTION
**Pull request by officialfrancismendoza**
**Issue (#250)**: No documentation regarding start_sequence in Clio config file
**Fix**: Update documentation regarding optional start_sequence parameter

### CODE REVIEW

**Review by godexsoft — COMMENTED** [review]
This text looks good.

I think we should also add it to `example-config.json` and maybe even add a brief comment explaining the same thing in the example config.

In `clio` configs, you can use `//` or `/* */` style comments in JSON.

**Comment by officialfrancismendoza** [comment]
@godexsoft left as additional comments under README.md since it's not possible to have pure comments in JSON files (data only). Could do a representation as a .toml file instead, but figured putting them in the documentation is a cleaner solution

File: README.md [hunk 1]
```diff
@@ -72,6 +72,21 @@ server is running
...
+- If this setting is present and the database is not empty, an exception is thrown.
+
+In addition, the optional parameter `finish_sequence` can be added to the json file as well, specifying where the ledger can stop.
+
+To add `start_sequence` and/or `finish_sequence` to the config.json file appropriately, they will be on the same top level of precedence as other parameters (such as database, etl_sources, read_only, etc.) and be specified with an integer. Here is an example below, added at the end of the config.json:
+
+```sh
+    ...
+    "read_only": false,
+    // "start_sequence": [integer] the ledger index to start from
```
Comment (godexsoft) [inline README.md]: In this case, when you are adding it to the readme like this, why keep the `//` for comments here?

File: README.md [hunk 2]
```diff
@@ -72,6 +72,21 @@ server is running
...
 
+In addition, the parameter `start_sequence` can be included and configured within the top level of the config file. This parameter specifies the sequence of first ledger to extract if the database is empty. Note that ETL extracts ledgers in order and that no backfilling functionality currently exists, meaning Clio will not retroactively learn ledgers older than the one you specify. Choosing to specify this or not will yield the following behavior:
+- If this setting is absent and the database is empty, ETL will start with the next ledger validated by the network. 
+- If this setting is present and the database is not empty, an exception is thrown.
+
+In addition, the optional parameter `finish_sequence` can be added to the json file as well, specifying where the ledger can stop.
+
+To add `start_sequence` and/or `finish_sequence` to the config.json file appropriately, they will be on the same top level of precedence as other parameters (such as database, etl_sources, read_only, etc.) and be specified with an integer. Here is an example below, added at the end of the config.json:
+
+```sh
```
Comment (godexsoft) [inline README.md]: Is there no `json` variant for syntax highlighting?

**Review by godexsoft — CHANGES_REQUESTED** [review]
Leaving a few questions. 
Also, while the JSON spec does not allow comments, clio's parser of JSON config certainly allows comments. So you can add the comments to the example config too (as comments).

File: README.md (hunk 2 above)
Comment (officialfrancismendoza) [reply README.md]: @godexsoft not that I'm aware of for plaintext, but there is for code if encapsulated in ```sh ```to give the highlighting you would find on a CLI
image

Comment (godexsoft) [reply README.md]: Please try both `json` and then `javascript` and see if anything looks like actual json and not `shell` after syntax highlighting on github.

**Review by godexsoft — APPROVED** [review]

**Pull request merged**